### DISCUSSION TITLE
Implementing N2 Handover

### DISCUSSION THREAD

**Discussion by AliNawazish** [srsran/srsRAN_Project#175] [General Help]
I want to implement N2 Handover for my project in srsRAN 5G with Open5GS as the core network. I know Open5GS supports Handover, but I'm curious about how complete the srsRAN implementation is. I have seen NGAP messages like Handover Required and Command. So, I have two questions regarding this,
1. The srsRAN roadmap says Handover is coming soon. Is there any rough timeline for it?
2. What is yet to implement in the Handover procedure? Is it just the RRC reconfiguration procedure that needs to be added? I would love to have a rough idea of this.

Thanks.

**Comment by pgawlowicz** [comment]
Currently, we are working on the intra-gnb handover, i.e., between two cells within the same gnb. It will be released within next months.
